ci-operator/step-registry/ovn/conf/vsphere-dualstack/ovn-conf-vsphere-dualstack-commands.sh

```diff
@@ -35,3 +35,10 @@ networking:
   - 172.30.0.0/16
   - fd65:172:16::/112
 EOF
+
+if [[ "${IP_FAMILIES}" == "DualStackIPv6Primary" ]]; then
+  echo Swapping IP addresses
```

Comment:
I think you also need to swap order of `platform.vsphere.apiVIPs` and `platform.vsphere.ingressVIPs`